src/main/java/com/empowerhack/family/memberservice/Entity/Member.java

```diff
@@ -0,0 +1,31 @@
+package com.empowerhack.family.memberservice.Entity;
+
+import lombok.Data;
+
+import javax.persistence.*;
+import javax.validation.constraints.*;
+import java.util.Set;
+
+@Data
+@Entity
+final public class Member {
```

Comment:
Ok, based on this I was going to raise the private/protected

Rule of thumb, if it's not final, make the members protected unless you are 100% sure that an inheriting class should not modify it, in this case make it private with the relevant modifier methods. 

Just my opinion, which of course is correct :D